src/sass/layout/components/_skills-animate.scss

```diff
@@ -0,0 +1,48 @@
+.progress-html {
```

Comment:
czy ta animacja zostala gdzies uzyta? czy to jest animacja uzyta w progress-bar?